Modules/ITS/itsQCTrending.json

```diff
@@ -0,0 +1,719 @@
+            "moduleNames": ["QcCommon","QcCommon"]
+          },
+          {
+            "types": ["repository","repository"],
+            "paths": ["qc/ITS/ITSTHRTask2B/Threshold/Layer2/Stave14/Threshold","qc/ITS/ITSTHRTask2B/DeadPixel/Layer2/Stave14/DeadPixelHITMAP"],
+            "names": [ "Threshold_l2_14", "DeadPixels_l2_14" ],
+            "reductorNames": ["o2::quality_control_modules::common::TH1Reductor", "o2::quality_control_modules::common::THnSparseReductor"],
+            "moduleNames": ["QcCommon","QcCommon"]
+          },
+          {
+            "types": ["repository","repository"],
+            "paths": ["qc/ITS/ITSTHRTask2B/Threshold/Layer2/Stave15/Threshold","qc/ITS/ITSTHRTask2B/DeadPixel/Layer2/Stave15/DeadPixelHITMAP"],
+            "names": [ "Threshold_l2_15", "DeadPixels_l2_15" ],
+            "reductorNames": ["o2::quality_control_modules::common::TH1Reductor", "o2::quality_control_modules::common::THnSparseReductor"],
+            "moduleNames": ["QcCommon","QcCommon"]
+          },
+          {
+            "types": ["repository","repository"],
+            "paths": ["qc/ITS/ITSTHRTask2B/Threshold/Layer2/Stave16/Threshold","qc/ITS/ITSTHRTask2B/DeadPixel/Layer2/Stave16/DeadPixelHITMAP"],
+            "names": [ "Threshold_l2_16", "DeadPixels_l2_16" ],
+            "reductorNames": ["o2::quality_control_modules::common::TH1Reductor", "o2::quality_control_modules::common::THnSparseReductor"],
+            "moduleNames": ["QcCommon","QcCommon"]
+          },
+          {
+            "types": ["repository","repository"],
+            "paths": ["qc/ITS/ITSTHRTask2B/Threshold/Layer2/Stave17/Threshold","qc/ITS/ITSTHRTask2B/DeadPixel/Layer2/Stave17/DeadPixelHITMAP"],
+            "names": [ "Threshold_l2_17", "DeadPixels_l2_17" ],
+            "reductorNames": ["o2::quality_control_modules::common::TH1Reductor", "o2::quality_control_modules::common::THnSparseReductor"],
+            "moduleNames": ["QcCommon","QcCommon"]
+          },
+          {
+            "types": ["repository","repository"],
+            "paths": ["qc/ITS/ITSTHRTask2B/Threshold/Layer2/Stave18/Threshold","qc/ITS/ITSTHRTask2B/DeadPixel/Layer2/Stave18/DeadPixelHITMAP"],
+            "names": [ "Threshold_l2_18", "DeadPixels_l2_18" ],
+            "reductorNames": ["o2::quality_control_modules::common::TH1Reductor", "o2::quality_control_modules::common::THnSparseReductor"],
+            "moduleNames": ["QcCommon","QcCommon"]
+          },
+          {
+            "types": ["repository","repository"],
+            "paths": ["qc/ITS/ITSTHRTask2B/Threshold/Layer2/Stave19/Threshold","qc/ITS/ITSTHRTask2B/DeadPixel/Layer2/Stave19/DeadPixelHITMAP"],
+            "names": [ "Threshold_l2_19", "DeadPixels_l2_19" ],
+            "reductorNames": ["o2::quality_control_modules::common::TH1Reductor", "o2::quality_control_modules::common::THnSparseReductor"],
+            "moduleNames": ["QcCommon","QcCommon"]
+          }
+        ],
+        "plots": [
```

Comment:
Regardless of the potential improvement for dataSources, I don't see how to improve it similarly for plots with the current version of TrendingTask.
I will have to see in August if there might be some way of creating multiple very similar plots without too much boilerplate

Reply:
Thanks

Reply:
Yes, I will have also to think about in the next days.